build_scripts/start_course.sh

```diff
@@ -9,20 +9,30 @@ else
   exit 1
 fi
 
+# If the DOWNLOAD variable isn't set, default it to 0
+if [[ -z "${DOWNLOAD+x}" ]]; then
+  DOWNLOAD=1
+fi
+
 if [[ -z "${OCW_TEST_COURSE}" ]]; then
   echo "OCW_TEST_COURSE not set"
   exit 1
 else
   rm -rf private/ocw-to-hugo-output/$OCW_TEST_COURSE
   mkdir -p private
   rm -f private/test_course.json
-  echo "{\"courses\":[\"${OCW_TEST_COURSE}\"]}" | tee private/test_course.json
-  /bin/bash build_scripts/import_courses.sh  private/test_course.json
+  if [[ $DOWNLOAD == 1 ]]; then
+    echo "{\"courses\":[\"${OCW_TEST_COURSE}\"]}" | tee private/test_course.json
+    /bin/bash build_scripts/import_courses.sh  private/test_course.json
+  else
+    rm -rf private/ocw-to-hugo-output/$OCW_TEST_COURSE
+    cp -r $OCW_TO_HUGO_OUTPUT_DIR/$OCW_TEST_COURSE private/ocw-to-hugo-output/
+  fi
   cp example_configs/example.course.go.mod private/ocw-to-hugo-output/$OCW_TEST_COURSE/go.mod
   cp example_configs/example.course.config.toml private/ocw-to-hugo-output/$OCW_TEST_COURSE/config/_default/config.toml
   mkdir -p private/ocw-to-hugo-output/$OCW_TEST_COURSE/static/pdfjs
   cp -r www/static/pdfjs/* private/ocw-to-hugo-output/$OCW_TEST_COURSE/static/pdfjs
   cd private/ocw-to-hugo-output/$OCW_TEST_COURSE
   hugo mod get -u
-  hugo server -p 3000 --bind 0.0.0.0 --renderToDisk
+  hugo server -p 3000 --bind 0.0.0.0 --renderToDisk --gc
```

Comment:
What does `--gc` do here? Should we have this flag in other scripts too?

Reply:
It forces garbage collection and was just something I was using to troubleshoot.  I don't think it's necessary, as my issue was elsewhere.  This kind of all sprang from testing out the "other versions" functionality.  Since a reverse lookup needs to be made on `master_subject` if you only convert one course in `ocw-to-hugo` and run that locally, then `ocw-to-hugo` sees the master subject set on the course and generates the other versions section, but no other versions appear because `ocw-to-hugo` doesn't have the context of the other courses.  Being able to point to an already converted external course would allow you to convert a batch of courses and then point at one specifically from that batch to spin up so things like the master subject reverse lookup function properly.